src/ui/units/dash/containers/Dialogs/DialogRelations/components/Content/Row.tsx

```diff
@@ -158,34 +160,62 @@ export const getTooltipInfo = ({
         relationType,
     });
 
-    const widgetLabel =
-        widget?.label && widget?.title !== widget?.label ? `${widget?.label} — ` : '';
-    const rowLabel = row?.label && row?.title !== row?.label ? `${row?.label} — ` : '';
+    const widgetLabel = widget?.label && widget?.title !== widget?.label ? `${widget?.label}` : '';
+    const rowLabel = row?.label && row?.title !== row?.label ? `${row?.label}` : '';
+
+    const tooltipWidgetTitle = getClampedText(widget.title);
+    const tooltipRowTitle = getClampedText(row.title);
+
+    const tooltipWidgetSeparator = getTextSeparator(widgetLabel, tooltipWidgetTitle);
+    const tooltipRowSeparator = getTextSeparator(rowLabel, tooltipRowTitle);
 
     const tooltipContent = (
         <React.Fragment>
             {getRelationDetailsText({
                 text: i18n(getRelationDetailsKey(relationType)),
                 linkType: relationType as RelationType,
-                widget: `${widgetLabel} ${getClampedText(widget.title)}`,
-                row: `${rowLabel} ${getClampedText(row.title)}`,
+                widget: `${widgetLabel}${tooltipWidgetSeparator}${tooltipWidgetTitle}`,
+                row: `${rowLabel}${tooltipRowSeparator}${tooltipRowTitle}`,
                 withStrong: true,
             })}
             {showByField && fieldTextWithStrong}
         </React.Fragment>
     );
 
+    const widgetTitle = widget.title;
+    const rowTitle = row.title;
+
+    const widgetSeparator = getTextSeparator(widgetLabel, widgetTitle);
+    const rowSeparator = getTextSeparator(rowLabel, rowTitle);
+
     const tooltipTitle =
-        widget.title?.length > TEXT_LIMIT || row.title?.length > TEXT_LIMIT
+        widget.title?.length > TEXT_LIMIT || rowTitle?.length > TEXT_LIMIT
             ? getRelationDetailsText({
                   text: i18n(getRelationDetailsKey(relationType)),
                   linkType: relationType as RelationType,
-                  widget: `${widgetLabel} ${widget.title}`,
-                  row: `${rowLabel} ${row.title}`,
+                  widget: `${widgetLabel}${widgetSeparator}${widgetTitle}`,
+                  row: `${rowLabel}${rowSeparator}${rowTitle}`,
               }) + (showByField ? fieldText : '')
             : undefined;
 
-    return {tooltipContent, tooltipTitle};
+    const aliasDetailTitle = (
+        <React.Fragment>
+            {getRelationDetailsText({
+                text: i18n(getRelationDetailsKey(relationType)),
+                linkType: relationType as RelationType,
+                widget: `${widgetLabel}${widgetSeparator}${widgetTitle}`,
+                row: `${rowLabel}${rowSeparator}${rowTitle}`,
```

Comment:
Row and widget are identical here and in `tooltipTitle`. Perhaps we should put this in a variable?

Reply:
It is not final variant of texts, I'm sure after design review we will change it not once